package.json

```diff
@@ -28,6 +28,9 @@
     "test": "npm run lint && npm run unit-test",
     "unit-test": "mocha tests/**/*.js"
   },
+  "peerDependencies": {
+    "eslint": ">=2.0.0"
```

Comment:
we might want to be explicit here and exclude anything above v6.x because we don't know if it will actually work 🤔

Reply:
Good point, especially if you can be responsive when it's time to support higher versions. :)

I've made the change.